tests/dedicatedvstests/l7_dedicated_graph_test.go

```diff
@@ -216,6 +216,29 @@ func SetUpIngressForCacheSyncCheck(t *testing.T, ingTestObj IngressTestObject) {
 	integrationtest.PollForCompletion(t, ingTestObj.modelNames[0], 5)
 }
 
+func SetUpTestForIngressInNodePortMode(t *testing.T, svcName, model_Name string) {
+	objects.SharedAviGraphLister().Delete(model_Name)
+	integrationtest.CreateSVC(t, "default", svcName, corev1.ProtocolTCP, corev1.ServiceTypeNodePort, false)
+}
+
+func TearDownTestForIngressInNodePortMode(t *testing.T, svcName, model_Name string) {
+	objects.SharedAviGraphLister().Delete(model_Name)
+	integrationtest.DelSVC(t, "default", svcName)
+}
+
+func VerifyIngressDeletion(t *testing.T, g *gomega.WithT, aviModel interface{}, poolCount int) {
+	var nodes []*avinodes.AviVsNode
+	g.Eventually(func() []*avinodes.AviPoolNode {
```

Comment:
can we combine these conditions together?

Reply:
I tried but we cannot do it since Eventually function returns single gomega.AsyncAssertion.